### PR TITLE
Added senselessclay had60

### PR DESCRIPTION
**Pull request by senselessclay**
## Description

Added files for the had60. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/senselessclay/had60/info.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (drashna) [inline keyboards/senselessclay/had60/info.json:1]: This is not a valid info.json, and appears to be a via json file.  The via json files don't belong in this repo, and need to be PR'ed to the `the-via/keyboards` repo, instead. 

https://docs.qmk.fm/#/reference_info_json

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/senselessclay/had60/info.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    "keyboard_name": "had60",
+    "url": "https://senselessclay.com/",
+    "maintainer": "hadi",
+    "width": 15, 
+    "height": 5, 
+    "layouts": {
+      "LAYOUT_60_all": {
```
Comment (zvecr) [inline keyboards/senselessclay/had60/info.json:8]: This name does not exist within `had60.h`

File: keyboards/senselessclay/had60/had60.h [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define ____ KC_NO
+
+#define LAYOUT( \
```
Comment (zvecr) [inline keyboards/senselessclay/had60/had60.h:23]:
```suggestion
#define LAYOUT_all( \
```

Also iso/ansi LAYOUT macros should be added to remove the need to manually pad missing keys with KC_TRANS

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/senselessclay/had60/info.json (hunk 2 above)
Comment (senselessclay) [reply keyboards/senselessclay/had60/info.json]: Addressed!

File: keyboards/senselessclay/had60/had60.h (hunk 3 above)
Comment (senselessclay) [reply keyboards/senselessclay/had60/had60.h]: Made the suggested change, but having separate layouts is only a concern for the configurator, right? All positions within the matrix are accessible here, and some other merged boards also use a single configuration file despite having similar support.

**Comment by senselessclay** [comment]
@zvecr, made the changes! Can I get another review?

File: keyboards/senselessclay/had60/readme.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# had60
+
+![had60](https://i.imgur.com/Q0AtIAg.jpg)
```
Comment (fauxpark) [inline keyboards/senselessclay/had60/readme.md:3]:
```suggestion
![had60](https://i.imgur.com/Q0AtIAgh.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by senselessclay** [comment]
@fauxpark @zvec, do I need to do anything else to get this PR merged?

**Pull request merged**